ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/ConnectExecutorTest.java

```diff
@@ -243,6 +257,34 @@ public void shouldReturnErrorIfConnectorTypeIsEmpty() {
     assertThat(((ErrorEntity) entity.get()).getErrorMessage(), is(expectedError));
   }
 
+  @Test
+  public void shouldReturnPluggableErrors() {
```

Comment:
Thanks for the test coverage! I think it's worth adding separate unit tests for each of the three cases (403, 401, other). Hopefully it's quick.

Reply:
Done in https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4#diff-89adae5ad10f6b2fbed946a5e2cf663e25b4dc129525f3a0fff774428af3a08cR262-R344.